src/opnsense/mvc/app/models/OPNsense/Dnsmasq/Dnsmasq.xml

```diff
@@ -29,21 +32,45 @@
         <local_ttl type="IntegerField">
             <MinimumValue>0</MinimumValue>
         </local_ttl>
+        <no_dhcp_interface type="InterfaceField">
+            <Multiple>Y</Multiple>
+        </no_dhcp_interface>
+        <dhcp_fqdn type="BooleanField"/>
+        <dhcp_lease_max type="IntegerField">
+            <MinimumValue>0</MinimumValue>
+        </dhcp_lease_max>
+        <dhcp_authoritative type="BooleanField"/>
+        <dhcp_default_fw_rules type="BooleanField">
+            <Required>Y</Required>
+            <Default>1</Default>
+        </dhcp_default_fw_rules>
+        <no_ident type="BooleanField">
+            <Required>Y</Required>
+            <Default>1</Default>
+        </no_ident>
         <hosts type="ArrayField">
             <host type="HostnameField">
                 <IsDNSName>Y</IsDNSName>
                 <IpAllowed>N</IpAllowed>
-                <Required>Y</Required>
```

Comment:
should we drop this in 25.1.2 already? same for domain

Reply:
@fichtner it's only not required when a mac address is offered (which is not in 25.1.2)